src/test/java/seedu/address/testutil/TypicalPersons.java

```diff
@@ -33,12 +33,12 @@ public class TypicalPersons {
             .withAddress("123, Jurong West Ave 6, #08-111").withEmail("alice@example.com")
             .withPhone("94351253")
             .withAppointments("Physio:2024-12-01:1500-1600", "Orthopedic:2024-12-01:1200-1300")
-            .withTags("friends").build();
+            .withAllergies("friends").build();
     public static final Person BENSON = new PersonBuilder().withName("Benson Meier")
             .withNric("T0234547A").withDateOfBirth("2002-01-01").withGender("M")
             .withAddress("311, Clementi Ave 2, #02-25")
             .withEmail("johnd@example.com").withPhone("98765432")
-            .withTags("owesMoney", "friends")
+            .withAllergies("owesMoney", "friends")
```

Comment:
Try to change the parameters to actual allergies so we don't forget in the future